semantic-kernel/concepts/text-search/index.md

```diff
@@ -200,9 +200,88 @@ Next we recommend looking at [Text Search Abstractions](./text-search-abstractio
 ::: zone-end
 ::: zone pivot="programming-language-python"
 
-## Coming soon
+## Implementing RAG using web text search
 
-More coming soon.
+In the following sample code, you can choose between using Bing or Google to perform web search operations.
+
+> [!TIP]
+> Install the required packages using:
+>
+> `pip install semantic-kernel`
+> `pip install semantic-kernel-plugins-web`
+
+### Create text search instance
+
+Each sample creates a text search instance and then performs a search operation to get results for the provided query.
+
+#### Bing web search
+
+```python
+from semantic_kernel import Kernel
+from semantic_kernel.connectors.ai.open_ai import OpenAIChatCompletion
+from semantic_kernel.plugins.web import BingTextSearch
```

Comment:
Actually, the API that this uses is going to change, so probably better now to do Google for this sample.
Check https://github.com/microsoft/semantic-kernel/blob/main/python/samples/concepts/search/google_text_search_as_plugin.py for a much more complete code sample with google